app/experimenter/experiments/constants.py

```diff
@@ -365,6 +371,29 @@ class EmailType(models.TextChoices):
         Application.FOCUS_IOS: Version.FIREFOX_101,
     }
 
+    ROLLOUT_SUPPORT = (
```

Comment:
I think you want this to be a dictionary that maps application to version?

Reply:
yes I do!